lib/src/client.dart

```diff
@@ -621,6 +622,9 @@ class Client extends MatrixApi {
       final room = getRoomById(directChatRoomId);
```

Comment:
To get abandoned rooms here, you would have to make sure `loadArchiveWithTimeline` was called before

Reply:
No, abaondoned DM rooms are defined as rooms where you have joined but the other party has left the room